File: app/src/release/java/com/azizbekian/spyur/injection/BuildTypeModule.java
package com.azizbekian.spyur.injection;

import javax.inject.Singleton;

import dagger.Module;
import dagger.Provides;
import okhttp3.OkHttpClient;

/**
 * Created on July 29, 2016.
 *
 * @author Andranik Azizbekian (andranik.azizbekyan@gmail.com)
 */
@Module
public class BuildTypeModule {

    @Provides @Singleton
    public OkHttpClient provideOkHttpClient() {
        return new OkHttpClient.Builder().build();
    }
}

File: app/src/release/java/com/incentive/yellowpages/injection/module/BuildTypeModule.kt
package com.incentive.yellowpages.injection.module

import dagger.Module
import dagger.Provides
import okhttp3.OkHttpClient
import javax.inject.Singleton

@Module
class BuildTypeModule {

    @Provides
    @Singleton
    internal fun provideOkHttpClient(): OkHttpClient.Builder {
        return OkHttpClient.Builder()
    }

}
